web3.js/src/transaction.ts

```diff
@@ -204,6 +214,141 @@ export class Transaction {
    * Compile transaction data
    */
   compileMessage(): Message {
+    if (this._message) {
+      const originalTransaction = this._originalTransaction;
+      if (!originalTransaction) {
+        throw new Error(
+          'Transaction mutated after being populated from Message',
+        );
+      }
```

Comment:
This is just here to narrow the TypeScript type, right? If `_message` is non-null, then `_originalTransaction` is for sure non-null, isn't it?

Reply:
Removed in stringified version.